server/utils/DbColumnHelper.ts

```diff
@@ -0,0 +1,21 @@
+import { isPgsql } from '@server/datasource';
+import type { ColumnOptions, ColumnType } from 'typeorm';
+import { Column } from 'typeorm';
+
+const pgTypeMapping: { [key: string]: ColumnType } = {
+  datetime: 'timestamp with time zone',
```

Comment:
Can this be a timestamp without time zone to match how the datetime is stored in sqlite? I haven't tested that, so am not sure what the behaviour would be.

Reply:
I'll give it a try and make sure it works first

Reply:
Sqlite doesn't support that. In fact, that fork dropped sqlite completely

Reply:
[From the readme](https://github.com/Fat-Cats-LLC/overseerr?tab=readme-ov-file#current-features):
https://github.com/Fat-Cats-LLC/overseerr?tab=readme-ov-file#current-features

Reply:
My question was about Postgres actually. not using a different data type for Sqlite. 
More specifically, whether this can be done without any issues since imo it's always a good idea to store timestamps without the time zone:
```suggestion
  datetime: 'timestamp without time zone',
```